DiscordChatExporter.Cli/Commands/ExportGuildCommand.cs

```diff
@@ -60,8 +64,18 @@ await foreach (var channel in Discord.GetGuildChannelsAsync(GuildId, cancellatio
                 )
             )
             {
+                await console.Output.WriteAsync(
+                    $"  Found {progress++} threads.  {thread.Parent?.Name} - {thread.Name}"
+                        .PadRight(80)
+                        .Substring(0, 80) + '\r'
```

Comment:
Suggestion: instead of tracking progress separately, can just do `Found {channels.Count(c => c.IsThread)} threads`. Slower, but would avoid extra business logic (i.e. off-by-one errors are likely here) and wouldn't matter on top of already heavy I/O requests anyway.